packages/SystemUI/src/com/android/systemui/statusbar/policy/NetworkTraffic.java

```diff
@@ -290,7 +265,7 @@ private CharSequence formatOutput(long speed) {
                 unit = kunit;
                 formatSpeed =  decimalFormat.format(speed / (float)Kilo);
             } else {
-                decimalFormat = new DecimalFormat("0.00");
+                decimalFormat = new DecimalFormat("0.##");
```

Comment:
0.## is not desired. It will give "5." instead "5.00"

Reply:
It was for the case of 0 speeds. Instead of showing 0.00 KB/s it was preferred to have 0 KB/s. I'll implement a alternative way for achieving that.